src/main/resources/templates/credential-template.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
     <meta charset="UTF-8" />
-    <title>Identity Card</title>
+    <title>New Template Identity Card</title>
```

Comment:
revert this too as no change in template yet

Reply:
Okay Reverted.